TACC/Core-CMS-Resources
# feat(demdata_cms): delete a.k.a. migrate to core-cms-custom

## Overview

Migrate DemData code to https://github.com/TACC/Core-CMS-Custom/.

## Changes

- **deleted** `demdata_cms`

## Testing

Build and deploy CMS using https://github.com/TACC/Core-CMS-Custom/pull/164.

---

<details><summary>Archived</summary>

## Do Not Merge

This PR exists **only** to review diff of branch with which to build `demdata-cms` images.

## Overview

Add CSS from snippets in [(dev) TUP CMS](http://dev.tup.tacc.utexas.edu/) (from https://github.com/TACC/tup-ui).

## Related

- required by https://github.com/TACC/Core-CMS/pull/606
- clones templates from https://github.com/TACC/tup-ui/pull/174

## Changes

- add snippets, from http://dev.tup.tacc.utexas.edu/
- add templates, demdata-specific: `standard`, `fullwidth`

## Testing

0. Build CMS image for demdata-cms off of Core-CMS branch `/task/demdata-css-from-tup`.
1. Open localhost or https://prod.democratizing-site.tacc.utexas.edu/.
2. Verify resources load a `site.cms.css` **not** a `site.css`.
3. Verify two "Standard" and "Full Width" snippets are available, the second of each is "… (with CSS from TUP)".
4. Change any page to use either "…  (with CSS from TUP)" template.
5. Verify:
    - Styles change. (The more different content on the page, the more obvious the change.)
    - Core styles assets are loaded (e.g. `core-styles.base.css`, `core-styles.cms.css`).
    - **Many** `<style>` tags are loaded at the bottom of the `<head>`.

## UI

…

## Notes

About the snippets: https://github.com/TACC/Core-CMS-Resources/blob/task/demdata-css-from-tup/demdata-cms/templates/from-tup-cms/README.md

Progress toward a consistent programmatic way to support a light header: https://github.com/TACC/Core-Styles/pull/134

</details>
